src/Posts.jsx

```diff
@@ -210,16 +207,12 @@ query GetFollowingPosts($offset: Int, $limit: Int) {
 
 const loadMorePosts = () => {
   const queryName =
-    state.selectedTab == "following" && accountsFollowing
-      ? "GetFollowingPosts"
-      : "GetPostsQuery";
-  const type =
-    state.selectedTab == "following" && accountsFollowing ? "following" : "all";
+    state.selectedTab == "following" ? "GetFollowingPosts" : "GetPostsQuery";
```

Comment:
I don't really like that user without followed accounts will see the same posts from **"all"** tab but viewing **"following"** tab. It's confusing. This string enables filtering according to selected tab

Reply:
@shelegdmitriy how does this change the UX?

Reply:
Take a look at this [comment](https://github.com/near/near-discovery-components/pull/369#discussion_r1340735781) (the last one). If user doesn't have any followed account we'll show a message **"There is no any posts yet."**

Reply:
@TiffanyGYJ  We could use this as a an opportunity to point the user to where they can find people/communitites to follow.    Could you suggest some text and a link we can add here?   Something like,  "[Connect](https://near.org/people) and find people to follow to customize your activity feed."